docs/Configure-GenericAptRepo.md

```diff
@@ -0,0 +1,23 @@
+# How to configure a generic aptitude repo
+
+This is made on the latest ubuntu-xenial.
+
+## On the repo machine:
```

Comment:
What is a repo machine?

Add a 
## Requirements:
 Ubuntu Xenial 16.04 where the aptitude repo will be created and configured.
 Ubuntu Xenial 16.04 where apt will be configured to install packages from the previously configured repo.